src/scheduler/fifo.cpp

```diff
@@ -349,7 +349,7 @@ init_scheduling_cycle(status *policy, int pbs_sd, server_info *sinfo)
 				if (user != NULL) {
 					auto rj = find_resource_resv(sinfo->running_jobs, lj.name);
 
-					if (rj != NULL && rj->job != NULL) {
+					if (rj != NULL && rj->job != NULL && rj->job->is_prerunning != 1) {
```

Comment:
is_prerunning is a bool.  Say either !rj->job->is_prerunning, or rj->job->is_prerunning == false